### PR TITLE
Add `useHotkeysforCtrlAndMeta` hooks for save with shortcut

### PR DESCRIPTION
**Pull request by cihat**

### CODE REVIEW

**Comment by cihat** [comment]
@AykutSarac

File: src/containers/Editor/BottomBar.tsx [hunk 1]
```diff
@@ -115,6 +116,8 @@ export const BottomBar = () => {
     }
   }, [getJson, hasChanges, query.json, replace, setHasChanges, setVisible, user]);
 
+  useHotkeysforCtrlAndMeta(83, handleSaveJson);
```
Comment (AykutSarac) [inline src/containers/Editor/BottomBar.tsx:119]: move it to the line 77

**Review by AykutSarac — APPROVED** [review]

File: src/containers/Editor/BottomBar.tsx (hunk 1 above)
Comment (cihat-salik) [reply src/containers/Editor/BottomBar.tsx:119]: Hi @AykutSarac, i will do this tonight, is there any other point apart from this i will update.

Comment (cihat) [reply src/containers/Editor/BottomBar.tsx:119]: Hi @AykutSarac, i will do this tonight, is there any other point apart from this i will update.

Comment (cihat) [reply src/containers/Editor/BottomBar.tsx:119]: I can't move this line to 77 because it needs to define **handleSaveJson()** function before

**Comment by AykutSarac** [comment]
Thanks for the PR, as we moved to Mantine UI we no longer need this ❤️

**Pull request closed without merging**